### DISCUSSION TITLE
Both LlamaIndex and LangChain have ECMAScript Issues in VSCode

### DISCUSSION THREAD

**Discussion by hdulay** [microsoft/vscode-discussions#1277] [Extension Development QnA] [answered]
The workaround is to add `"type": "module",` in the package.json but this causes the extension to not load. Below is an example error:

> The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("llamaindex")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to '/Users/hubertdulay/development/hubert-vscode/obsequious/package.json'.ts(1479)

**Comment by isidorn** [comment, marked as answer]
I do not know how your are usign llamaIndex or LangChain in vscode, but if you are using some extension, the best would be to file an issue with that extension
